packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -77,21 +77,8 @@ export interface ViewableTree {
 	 * Only one schematized view may exist for a given ITree at a time.
 	 * If creating a second, the first must be disposed before calling `viewWith` again.
 	 *
-	 *
-	 * TODO: Provide a way to make a generic view schema for any document.
```

Comment:
This has been provided as an alpha API already, see generateSchemaFromSimpleSchema.